workflow-templates/im-test-k6-manual.yml

```diff
@@ -0,0 +1,78 @@
+name: Manual trigger K6 test
+
+on: 
+  workflow_dispatch:
+    inputs:
+      envToTest:
+        description: 'Environment to run test against [dev, qa, stage]'   
+        required: true
+        default: 'dev'
+      testFolderRoot:
```

Comment:
Something I noticed was your variables have kind of mixed casing.  Some use `_` between words and some use pascal casing.  You might want to change to one or the other for consistency.  The templates in this repo use `-` between words.

Reply:
Updating casing to match repo.